src/recordlinker/database/mpi_service.py

```diff
@@ -292,6 +306,23 @@ def update_person_cluster(
     return person
 
 
+def update_patient_person_ids(
+    session: orm.Session,
+    person: models.Person,
+    person_ids: typing.Sequence[int],
+    commit: bool = True,
+) -> models.Person:
+    """
+    Update the person_id for all Patients associated with a Person.
```

Comment:
Nit: could we update the language on this comment? The code itself seems to be updating to the ID of the provided person the person_ids of all patients currently associated to a person_id in the given list (a grotesque tongue twister, to be sure). This comment reads to me like we're going to be passing in a list of patients and then just batch setting them all to point to the given person ID. Not blocking by any means, I just think the explicit clarity would reduce the double takes.

Reply:
This is ugly. What about "Update the person_id for all Patients by current person_id."?

Reply:
Yeah that's better (mine was hideous I know 😅 ; was just trying to capture the actual logic of what was happening. I know the prepositional ordering of "updating _to the ID of the provided person_ the person_ids..." is grammatically correct but it always feels so wrong to write it out).